examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,20 @@
+''' A ac graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+by drawing three arcs of defined radius, defined start and 
+end angles at defined point(coordinates) .
+
+
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.arc
+    :refs: :ref:`ug_basic_lines_arcs`
+    :keywords: arc, figure, glyph
+'''
+
+
+
+
+
 from bokeh.plotting import figure, show
```

Comment:
nore spacing adjustments for consistency with other examples
```suggestion
    :keywords: arc, figure, glyph
    
'''
from bokeh.plotting import figure, show
```